addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10056,7 +10056,9 @@ msgctxt "#19239"
 msgid "Starting background threads"
 msgstr ""
 
-#empty string with id 19240
+msgctxt "#19240"
+msgid "RePlay"
```

Comment:
When adding strings, plase always provide subscriptive text (#.) and info in which file(s) the new string is used (#:)